### PR TITLE
a bit of cleanup of the new-ui

### PR DESCRIPTION
**Pull request by daveol**

### CODE REVIEW

**Comment by nomis52** [comment]
LGTM

File: javascript/new-src/src/js/app.js [hunk 1]
```diff
@@ -29,8 +29,6 @@ angular
   .factory('$ola', ['$http', '$window', 'OLA',
     function($http, $window, OLA) {
       'use strict';
-      // once olad supports json post data postEncode can go away
```
Comment (peternewman) [inline javascript/new-src/src/js/app.js]: We still need to do this don't we, so should leave the comment in, ideally with a TODO(Dave):

**Comment by peternewman** [comment]
LGTM too, just one comment

**Comment by peternewman** [comment]
LGTM, can you make it a todo please @daveol .

**Comment by peternewman** [comment]
LGTM

**Pull request merged**